app-arch/innoextract/innoextract-1.5.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils toolchain-funcs cmake-utils
+
+DESCRIPTION="A tool to unpack installers created by Inno Setup"
+HOMEPAGE="http://constexpr.org/innoextract/"
+SRC_URI="http://constexpr.org/innoextract/files/${P}.tar.gz"
+
+LICENSE="ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="c++0x debug doc +iconv +lzma static"
```

Comment:
No, you shouldn't ever explicitly check the GCC version. For example, because the user can be using a different compiler. If the configure script degrades gracefully, then it's cool.